ParkingLot.java

```diff
@@ -30,14 +30,14 @@ public ParkingLot(int parkingLotSize) {
 
     /**
      *Checks the status of a parking spot to see if there is a car there or not
-     *If the spot isn't valid then it throws an exception saying what values
+     *If the spot isn't valid then it throws an exception giving the valid options
      *@param parkingLotSpot is the parking spot that you are getting the status on
      *@return returns false if the slot is full/car is parked there or true for empty/car is not parked there
      **/
     public boolean getStatus(int parkingLotSpot) throws Exception {
```

Comment:
since you've created a specific exception, you should update this "throws" to specify that exception type.